djhtml/__main__.py

```diff
@@ -1,210 +1,152 @@
-import argparse
-import sys
-from pathlib import Path
+"""
+Entrypoint for all 4 command-line tools. Typical usage:
 
-from . import modes
+    $ djhtml file1.html file2.html
 
+Passing "-" as the filename will read from standard input and write to
+standard output. Example usage:
 
-def verify_changed(source, result):
-    """
-    Verify that the source is either exactly equal to the result or
-    that the result has only changed by added or removed whitespace.
+    $ djhtml - < input.html > output.html
+"""
 
-    """
-    output_lines = result.split("\n")
-    changed = False
-    for line_nr, line in enumerate(source.split("\n")):
-        if line != output_lines[line_nr]:
-            changed = True
-        if line.strip() != output_lines[line_nr].strip():
-            raise IndentationError("Non-whitespace changes detected. Core dumped.")
+import sys
+from pathlib import Path
 
-    return changed
+from . import modes, options
 
 
 def main():
-    """
-    Entrypoint for all 4 command-line tools. Typical usage:
-
-        $ djhtml -i file1.html file2.html
+    changed_files = 0
+    unchanged_files = 0
+    problematic_files = 0
 
-    """
-    target_extension = ".html"
+    suffixes = [".html"]
     Mode = modes.DjHTML
     if sys.argv[0].endswith("djtxt"):
         Mode = modes.DjTXT
-        target_extension = ".txt"
+        suffixes = [".txt"]
     if sys.argv[0].endswith("djcss"):
         Mode = modes.DjCSS
-        target_extension = ".css"
+        suffixes = [".css", ".scss"]
```

Comment:
These are hardcoded and not configurable, so I figured they should be at least equal to the default pre-commit hook, which processes both `css` and `scss` files.